### PR TITLE
fix!: fix dashboard chart name

### PR DESCRIPTION
**Pull request by Wielewout**
This PR renames the `dash-front-back-end` chart to just `dashboard`.

### CODE REVIEW

**Comment by Wielewout** [comment]
This PR should be "rebased and merged" btw as the separate commit messages should be shown in the changelog.

File: charts/drax/templates/service-smo-api.yaml
Comment (ben-foxmoore) [inline charts/drax/templates/service-smo-api.yaml:1]: I need to take some time to understand what's going on here. If we want to merge the rest of this immediately for a new  7.0 RC, could we split it into two PRs?

Comment (Wielewout) [reply charts/drax/templates/service-smo-api.yaml:1]: Sure, I'll take the init container out.

Comment (ben-foxmoore) [reply charts/drax/templates/service-smo-api.yaml:1]: Maybe we can do the rename too? At least that way, adding the SMO API service later doesn't break the values file format.

Comment (Wielewout) [reply charts/drax/templates/service-smo-api.yaml:1]: I'm gonna split it up with all 3 original commits in a separate PR.
This one will be the dashboard renaming then :smile:

**Review by ben-foxmoore — APPROVED** [review]
PR description needs to be updated (mentions new SMO API) if you're going to use Github's rebase/squash merge, but otherwise LGTM

**Pull request merged**